portfolio/src/main/webapp/style.css

```diff
@@ -23,3 +45,80 @@
 .image-thumbnails img:hover {
   border: thick solid LIGHTCORAL;
 }
+
+
+
+body, html {
```

Comment:
do you need html, too?

Reply:
Yeah so that the pictures on the photography page at the top display.